tensorflow/compiler/mlir/tensorflow/transforms/lower_tf.cc

```diff
@@ -113,12 +113,27 @@ Type InferExpandDimsType(Type ty, int64_t axis, Builder *builder) {
 
 // Lowers AddN op to a sequence of AddV2 ops to accumulate operands.
 //
+// Note that to improve the parallelism, the operands are split
+// into two halves, and are accumulated first.
+//
+// Example:
+//
 //   %result = "tf.AddN"(%0, %1, %2)
 //
 // is lowered to:
 //
-//   %sum_0 = "tf.AddV2"(%0, %1)
-//   %result = "tf.AddV2"(%sum_0, %2)
+//   %sum_right = "tf.AddV2"(%1, %2)
+//   %result = "tf.AddV2"(%0, %sum_right)
+//
+// Or
```

Comment:
s/Or/While/ ? (I read the or to be a clause on "is lowered to" instead of separate sentence, and this made this less likely to misread)